### ISSUE TITLE
Build for Android and IOS

### ISSUE THREAD

**Issue by darkyelox** [bbc/audiowaveform#79]
Hello, i want to use this great library inside my NativeScript app by a plugin for generating .dat files from an MP3 input, but don't know how to compile, and if its posible, for Android and IOS systems, would be great have a clue and i do the work and maybe do a PR.

The idea is create and executable that can be bundled within a plugin and executed as a shell command.

**Comment by NelsonJTSM** [comment]
@darkyelox Were you ever able to compile to Android/iOS?

**Comment by darkyelox** [comment]
> @darkyelox Were you ever able to compile to Android/iOS?

No, sadly. I had to create my own implementation using a canvas library in nativescript but using a predefined shape instead of getting the sound data
